cpp/ql/src/Likely Bugs/Arithmetic/SignedOverflowCheck.qhelp

```diff
@@ -4,26 +4,33 @@
 <qhelp>
 <overview>
 <p>
-Testing for signed integer overflow by adding a
-two signed values together and then comparing the result to one
-of the values is ill-formed since the overflow check is undefined.
-The comparison may produce an unintended result, or may be deleted
-by the compiler entirely.
+When checking for integer overflow, one often writes tests like
+<code>a + b &lt; a</code>.  This works fine if <code>a</code> or
+<code>b</code> are unsigned integers, since any overflow in the addition
+will cause the value to simply "wrap around".  However, using
```

Comment:
Use US-style punctuation, with the closing full stop inside the closing quote:

simply "wrap around."

Reply:
Do Brits do it differently? :-)